tests/system/src/test/java/test/robot/scenegraph/JDK8130122Test.java

```diff
@@ -98,9 +99,10 @@ public void testEmptyShapes() {
             assertColorEquals(Color.WHITE, color, TOLERANCE);
             horizontalListView.setVisible(true);
         });
+        sleep(1000);
```

Comment:
we still have JDK-8176902 open, but perhaps here we could add a new method to VisualTestBase similar to waitNextFrame(), with an appropriate parameter?

Something like
```
protected void waitForIdle() {
  frameWait(100);
}
```

instead of 1 second sleep() ?

Reply:
I think waitForIdle() would not be a proper name for this use case.
And we already have waitFirstFrame() which does frameWait(100).

What i will do is, use waitFirstFrame() and add comment in test case that we are waiting after setVisible(true).